src/front-end/typescript/lib/app/router.ts

```diff
@@ -31,7 +31,73 @@ export function back() {
 
 const router: Router<Route> = {
 
+  // Note(Jesse): @add_new_page_location
+
   routes: [
+
+    {
+      path: '/code-with-us/opportunities/create',
+      makeRoute() {
+        return {
+          tag: 'cwuOpportunityCreate',
+          value: null
+        };
+      }
+    },
+    {
+      path: '/code-with-us/opportunities/edit',
+      makeRoute() {
+        return {
+          tag: 'cwuOpportunityEdit',
+          value: null
+        };
+      }
+    },
+    {
+      path: '/code-with-us/opportunities/view',
```

Comment:
Update to `/code-with-us/opportunities/:opportunityId` (add ID & remove the `view` string) and update the page to support an `opportunityId` route param